forge.config.js

```diff
@@ -29,6 +30,7 @@ export default {
       config: {
         options: {
           icon: "assets/icons/icon.png",
+          format: "ULFO",
```

Comment:
@RobertGemmaJr we can delete this line! I added this in but it doesn't seem to do anything

Reply:
```suggestion
```